src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -188,13 +177,12 @@ object WorkflowDescriptor {
     }
   }
 
-  private def validateBackendOptions(id: WorkflowId, options: WorkflowOptions, backend: Backend): ErrorOr[WorkflowOptions] = {
+  private def validateBackendOptions(id: WorkflowId, options: WorkflowOptions): ErrorOr[WorkflowOptions] = {
     try {
-      backend.assertWorkflowOptions(options)
       options.successNel
     } catch {
       case e: Exception =>
-        s"Workflow $id has invalid options for backend ${backend.backendType}: ${e.getMessage}".failureNel
+        s"Workflow $id has invalid options: ${e.getMessage}".failureNel
```

Comment:
We can talk more about where and how it should live, but from a end-user functionality perspective it needs to be accessible at the point a WDL is received by the server in a fail-immediately fashion - both from a workflow submission and validation (which are currently 2 separate code paths, but transforming into a single actor type to handle it). Any implementation scheme must be robust to that.

If a call might be able to be sent to multiple backends that's slightly trickier, w/o having thought about it deeply I'd imagine something where as long as a call had a shape that could fit in *some* hole that it'd be fine. The error would be if it couldn't fit in any backend hole.